website/docs/reference/deprecations.md

```diff
@@ -2,6 +2,12 @@
 title: "Deprecations"
 ---
 
+:::note
```

Comment:
open to other formatting here!

Reply:
trying to echo the sentiment from the [deprecations warning github post](https://github.com/dbt-labs/dbt-core/discussions/11493):

> We know it will take existing users time to migrate their projects, and it is not our goal to break anyone overnight. The goal is to enable dbt users to work with more safety, feedback, and confidence going forward.